python/prairielearn.py

```diff
@@ -162,6 +181,8 @@ def from_json(v):
                     return pandas.DataFrame(index=val['index'], columns=val['columns'], data=val['data'])
                 else:
                     raise Exception('variable of type dataframe should have value with index, columns, and data')
+            elif v['_type'] == 'dataframe-v2':
+                return pandas.read_json(v['_value'], orient="table")
```

Comment:
```suggestion


                # Convert native JSON back to a string representation so that 
                # pandas read_json() can process it.
                value_str = json.dumps(v['_value'])
                return pandas.read_json(value_str, orient="table")
```

Reply:
@coatless Added the changes you suggested and tests + element seem to be working!